src/include/update/rootpart.rb

```diff
@@ -77,6 +77,28 @@ def CanBeLinuxRootFS(partition_fs)
       end
     end
 
+    # Check if partitions_fs was a valid filesystem for '/' mount point
+    #
+    # @param partition_fs [Symbol]
+    #
+    # @return [Boolean]
+    #
+    # @example
+    #    ok = legacy_filesystem?(:reiserfs)
+    #
+    def legacy_filesystem?(partition_fs)
+      if partition_fs.nil?
+        Builtins.y2error("partition_fs not defined!")
+        return false
+      end
+
+      begin
+        Y2Storage::Filesystems::Type.new(partition_fs).legacy_root?
+      rescue
+        false
```

Comment:
I've seen the definition of `CanBeLinuxRootFS` and I guess you just got inspiration from it :)